### PR TITLE
Boot changes

### PR DESCRIPTION
**Pull request by Lamasmaster**
duty-jackboots
Removes regular Jackboots and replaces them with Knee and Thigh variants.
Buffs all boots up to same level as jungle/desert boots.

### CODE REVIEW

File: modular_boh/code/items/clothing/boh_clothing.dm [hunk 1]
```diff
@@ -148,4 +148,19 @@
 	icon_state = "labcoat_cso_open"
 	icon_open = "labcoat_cso_open"
 	icon_closed = "labcoat_cso"
-	body_parts_covered = UPPER_TORSO|ARMS
\ No newline at end of file
+	body_parts_covered = UPPER_TORSO|ARMS
+
+//#### Boots
+/obj/item/clothing/shoes/dutyboots/knee
+	name = "jackboots - knee"
```
Comment (BurpleBineapple) [inline modular_boh/code/items/clothing/boh_clothing.dm]:
```suggestion
	name = "knee high jackboots"
```

File: modular_boh/code/items/clothing/boh_clothing.dm [hunk 2]
```diff
@@ -148,4 +148,19 @@
...
+	desc = "Tall synthleather boots with an artificial shine."
+	icon = 'icons/obj/clothing/obj_feet.dmi'
+	icon_state = "jackboots_knee"
+	item_state = "jackboots_knee"
+
+/obj/item/clothing/shoes/dutyboots/thigh
+	name = "jackboots - thigh"
```
Comment (BurpleBineapple) [inline modular_boh/code/items/clothing/boh_clothing.dm]:
```suggestion
	name = "thigh high jackboots"
```

**Review by BurpleBineapple — CHANGES_REQUESTED** [review]

**Review by Superhats — APPROVED** [review]
Good stuff

**Review by BurpleBineapple — APPROVED** [review]
Works

**Pull request merged**